app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,10 @@
+class MoviesController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+    if params[:top]
+      @movies_facade = MoviesFacade.new.top_rated
+    else
+      @movies_facade = MoviesFacade.new.search(params[:query])
+    end
```

Comment:
This conditional is great to utilize the index to route to the two different endpoints through the facade.